atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -253,7 +252,18 @@ public void compactInternally() {
                         + " IOT tables to compensate for bloat. You can contact Palantir Support if you'd"
                         + " like more information. Underlying error was: {}", tableRef, e.getMessage());
             } catch (TableMappingNotFoundException e) {
-                throw Throwables.propagate(e);
+                throw new RuntimeException(e);
+            }
+        } else {
+            try {
+                log.warn("Tried to clean up table {}, but couldn't, because enterprise features are disabled. "
```

Comment:
Do we need to keep the whole existing message the same? I find the second part of the new log message more informative - it actually says which command to run, for instance. If the "advice" parts should be different for the two messages, then there really isn't much in common between the two log messages, so I don't see much value in extracting the common parts.